pkg/cmds/cli/download.go

```diff
@@ -0,0 +1,106 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/appscode/go/flags"
+	"github.com/appscode/go/log"
+	"github.com/appscode/stash/pkg/restic"
+	"github.com/appscode/stash/pkg/util"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewDownloadCmd() *cobra.Command {
+	var (
+		kubeConfig     string
+		repositoryName string
+		namespace      string
+		destination    string
+		restoreOpt     = restic.RestoreOptions{
+			SourceHost: restic.DefaultHost,
+		}
+	)
+
+	var cmd = &cobra.Command{
+		Use:               "download",
+		Short:             `Download snapshots`,
+		Long:              `Download contents of snapshots from Repository`,
+		DisableAutoGenTag: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			flags.EnsureRequiredFlags(cmd, "repository", "destination")
```

Comment:
Can't we make `destination` flag optional? It can be defaulted to present directory.